### PR TITLE
Pin Jinja2 and Markupsafe libraries

### PR DESCRIPTION
**Pull request by rolisz**
The latest version of these libraries is not compatible with eli5

### CODE REVIEW

**Comment by Elypha** [comment]
I have the same issue upon import eli5 and can confirm it on a clean installation of both python39 and python310. Your fix works well for me so far and hope it could be merged as a temporary solution

**Review by lopuhin — COMMENTED** [review]
Thanks for the PR @rolisz , I think the libraries also need to be updated here as well https://github.com/eli5-org/eli5/blob/69637074ad07cdb0d0aa3a650302a721c9272e4b/setup.py#L33-L41 - I can pick that up from here.

**Comment by lopuhin** [comment]
Thanks for the PR, this was closed in https://github.com/eli5-org/eli5/pull/19
